libraries/lib-time-and-pitch/StaffPad/VectorOps.h

```diff
@@ -128,7 +121,7 @@ inline void rotate(
 inline void calcPhases(const std::complex<float>* src, float* dst, int32_t n)
```

Comment:
I would like to see an indication here that it's not a straightforward phase calculation of the `src` vector. I'm thinking renaming `calcPhasesAlternating`, consistently with the naming elsewhere, and also adding a comment where it is used in TimeAndPitch.cpp.

Reply:
A little bit of pseudo maths in the comment explaining the mathematical equivalence would also be welcome.